src/iap/shared/modules/api/services/user.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { Observable, map } from 'rxjs';
+
+import { User } from '@shared/models/entities/user.entity';
+import { Profile } from '@shared/models/entities/profile.entity';
+import { UpdateProfileDto } from '@shared/models/dto/update-profile.dto';
+import { ApiService } from '@shared/services/api.service';
+
+@Injectable()
+export class UserService {
+	constructor(private api: ApiService) {}
+
+	get(id: number): Observable<User> {
+		const url = `user/${id}`;
+		return this.api.get(url).pipe(map((user: any) => new User(user?.id, user?.login)));
```

Comment:
I believe it's a good idea to use special models to show what type of data comes from the backend:
```suggestion
	get(id: number): Observable<User> {
		const url = `user/${id}`;
		return this.api.get(url).pipe(map((user: BackendUser) => new User(user?.id, user?.login)));
```